fast-af/fast-checkout-woocommerce
# Update prefixes

# Description

Update function and constant prefixes to be more specific to the plugin. This is due to the feedback from the WordPress Plugin Directory review:

# Testing instructions

1. Install the update.
2. Verify that everything continues to work as it did before.

# Checks
